pkgs/by-name/ad/adbtuifm/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  android-tools,
+  lib,
+}:
+buildGoModule rec {
+  pname = "adbtuifm";
+  version = "0.5.8";
+  src = fetchFromGitHub {
+    owner = "darkhz";
+    repo = "adbtuifm";
+    rev = "v${version}";
+    hash = "sha256-TK93O9XwMrsrQT3EG0969HYMtYkK0a4PzG9FSTqHxAY=";
+  };
+  vendorHash = "sha256-voVoowjM90OGWXF4REEevO8XEzT7azRYiDay4bnGBks=";
+  buildInputs = [
+    android-tools
```

Comment:
this is not a buildInput as adbtuifm builds without android-tools

I assume it's runtime dependency ? you can either expect user to install it manually or use wrapProgram

https://nixos.org/manual/nixpkgs/unstable/#fun-wrapProgram

sorry for adhoc feedback, i should have had a popper look earlier

Reply:
Actually, after inspecting the [adb.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/programs/adb.nix) module I came to realize that it's better if the user manually installs adb since it needs proper configuration to work